### PR TITLE
[NCL-8471] Add option to upload logs to bifrost

### PR DESCRIPTION
**Pull request by janinko**

### CODE REVIEW

File: common/src/main/java/org/jboss/pnc/buildagent/common/security/Checksum.java [hunk 1]
```diff
@@ -17,37 +17,40 @@
...
 
 /**
  * @author <a href="mailto:matejonnet@gmail.com">Matej Lazar</a>
  */
-public class Md5 {
+public class Checksum {
 
-    private MessageDigest md;
+    private MessageDigest md5;
+    private MessageDigest sha512;
```
Comment (matejonnet) [inline common/src/main/java/org/jboss/pnc/buildagent/common/security/Checksum.java:31]: Why the two checksums for the logs? The initial idea was to make sure the log is completely transferred, I picked MD5 which is faster than SHA, I was event thinking to use CRC.

Comment (janinko) [reply common/src/main/java/org/jboss/pnc/buildagent/common/security/Checksum.java:31]: @thescouser89 ^

Comment (thescouser89) [reply common/src/main/java/org/jboss/pnc/buildagent/common/security/Checksum.java:31]: Hey sure! If it's only used to very if the log is completely transferred, then md5 is good enough.

Woops @janinko  I've probably created more work than needed to you. Sorry!

**Comment by thescouser89** [comment]
retest this please

**Comment by janinko** [comment]
retest this please

**Pull request merged**